pages/task/list/content/index.js

```diff
@@ -55,6 +55,15 @@ module.exports = {
         ra: 'Retrospective assessments',
         changeLicenceHolder: 'Change licence holder'
       }
+    },
+    pelType: {
+      label: 'By PEL type:',
+      options: {
+        places: 'Schedule of premises',
+        roles: 'Named roles',
```

Comment:
Elsewhere we refer to these as "Approved areas" and "Named people". We should to be consistent in that.